### DISCUSSION TITLE
Flameshot not opening on Fedora Linux 37

### DISCUSSION THREAD

**Discussion by ekaser** [flameshot-org/flameshot#3087] [General]
Hi all,

I've tried installing the latest release of Flameshot from both the integrated software store and SnapCraft, to no avail. When I install and attempt to open Flameshot, nothing happens.

I thought the problem might be Wayland, so I tried using Gnome X11, to no avail.

I'm wondering if anyone has insight or a similar experience? This seems to reflect a number of cases in the review section of the software store.

**Comment by mmahmoudian** [comment]
What happens when you run `flameshot gui` in terminal?

**Reply by ekaser** [reply]
Flameshot works as intended using that; thanks for this command!

**Reply by mmahmoudian** [reply]
Then your problem is because of the bad Gnome design. They removed the systray and as the result you don't see anything happening when running Flameshot (or any software that sits peacefully in systray and out of the way of user). Try either installing the correct gnome extension to get the systray, or bind your keyboard shortcut to `flameshot gui`, or ultimately use another DE (e.g KDE, Xfce, Budgie, ...)

**Comment by ijustwannathat** [comment]
Install flameshot via `flatpak` (software) and run this command `flatpak run org.flameshot.Flameshot gui`. That worked for me on Fedora 42
